src/lib/Components/Bidding/Screens/Registration.tsx

```diff
@@ -273,17 +274,24 @@ export class Registration extends React.Component<RegistrationProps, Registratio
           </View>
 
           <View>
-            <Checkbox mb={4} justifyContent="center" onPress={() => this.conditionsOfSalePressed()}>
+            <Checkbox
+              mb={4}
+              justifyContent="center"
+              onPress={() => this.conditionsOfSalePressed()}
+              disabled={isLoading}
+            >
               <Serif14 mt={2} color="black60">
-                Agree to <LinkText onPress={this.onPressConditionsOfSale}>Conditions of Sale</LinkText>.
+                Agree to[" "}
```

Comment:
I missed this during review, but there's a typo here that I've corrected here: https://github.com/artsy/emission/commit/1765d08643bd0bbd3d4f34d1524ea60d71dad633